packages/components/src/Notification/ToastNotification/_docs/ToastNotification.stories.tsx

```diff
@@ -76,6 +77,58 @@ export const CreateNotification: Story = {
   },
 }
 
+export const CreateSingleNotification: Story = {
+  render: () => {
+    const { addToastNotification, clearToastNotifications } =
+      useToastNotification()
+    return (
+      <>
+        <Button
+          label="Create notifications"
+          classNameOverride="!mr-12"
+          onClick={() => {
+            addToastNotification({
+              id: "id--clear-example-1",
+              title: "First",
+              type: "positive",
+              message: "There should only be one notification",
+            })
+            addToastNotification({
+              id: "id--clear-example-1",
+              title: "First",
+              type: "positive",
+              message: "There should only be one notification",
+            })
+            addToastNotification({
+              id: "id--clear-example-1",
+              title: "First",
+              type: "positive",
+              message: "There should only be one notification",
+            })
+          }}
+        />
+        <Button
+          label="Clear notifications"
+          onClick={() => clearToastNotifications()}
+        />
+      </>
+    )
```

Comment:
I would probably suggest you replicate your implementation with the `useEffect` instead if possible 🤞🏻 Only because this doesn't seem like someone would actually write this code, so it's easy to consider it an irrelevant test and delete it in the future